src/dialog-editor/services/modal/modalService.ts

```diff
@@ -81,6 +85,40 @@ class ModalController {
             }
           );
         }
+        // set modal title
+        if (!this.modalData.dynamic) {
+          var titleLabel;
+
+          switch (this.modalData.type) {
+            case 'DialogFieldTextBox':
+              titleLabel = __("Text Box");
+              break;
+            case 'DialogFieldTextAreaBox':
+              titleLabel = __("Text Area");
+              break;
+            case 'DialogFieldCheckBox':
+              titleLabel = __("Check Box");
+              break;
+            case 'DialogFieldDropDownList':
+              titleLabel = __("Dropdown");
+              break;
+            case 'DialogFieldRadioButton':
+              titleLabel = __("Radio Button");
+              break;
+            case 'DialogFieldDateControl':
+              titleLabel = __("Datepicker");
+              break;
+            case 'DialogFieldDateTimeControl':
+              titleLabel = __("Timepicker");
+              break;
+            case 'DialogFieldTagControl':
+              titleLabel = __("Tag Control");
+              break;
+          }
+          this.modalTitle =  __("Edit ") + titleLabel +  __(" Field");
```

Comment:
This is one way of creating strings in JS, however consider using [Template_literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
```Typescript
__(`Edit ${titleLabel} Field`);
```

Reply:
Hi, I'm not familiar enough with your translation tools.  I wasn't allowed to use a '+' in there, so I'm hesitant to use a template literal.